src/styles/blocks/card.scss

```diff
@@ -0,0 +1,80 @@
+@import '/src/styles/blocks/variables';
```

Comment:
Try to avoid using absolute paths in imports. It is better to use relative paths. Use `@import './variables';`